hasura/metadata/databases/default/tables/public_app.yaml

```diff
@@ -13,30 +13,25 @@ array_relationships:
         table:
           name: action
           schema: public
-computed_fields:
-  - name: is_verified
-    definition:
-      function:
-        name: get_app_is_verified
-        schema: public
-  - name: verified_app_logo
-    definition:
-      function:
-        name: get_verified_app_logo
-        schema: public
+  - name: app_metadata
+    using:
+      manual_configuration:
```

Comment:
why does this need `manual_configuration`? seems like it should work similar to how actions are linked above?

Reply:
This keeps suggesting an object relationship when we need an array one so I made it manually

Reply:
if you follow the same pattern as for actions and reload metadata?

Reply:
I tried to make it in the yaml directly but it then didn't show the relationship (still worked) in the console and thought it would be confusing to anyone else looking at it so I changed it back.